MapboxCoreNavigation/RouteController.swift

```diff
@@ -696,10 +709,7 @@ extension RouteController: CLLocationManagerDelegate {
             return true
         }
 
-        let radius = max(reroutingTolerance, RouteControllerManeuverZoneRadius)
-        let isCloseToCurrentStep = location.isWithin(radius, of: routeProgress.currentLegProgress.currentStep)
-
-        guard !isCloseToCurrentStep || !userCourseIsOnRoute(location) else { return true }
+        guard mbNavigatorStatus?.routeState == .offRoute || !userCourseIsOnRoute(location) else { return true }
```

Comment:
Cool! 

Could this be refactored to give us insight into occurrences where these values differ? I think that might be useful.